lib/supabase/mock.ts

```
@@ -0,0 +1,358 @@
+            data = mockSessions.find((s: any) => s[column] === value)
+          } else if (table === 'tasks') {
+            data = mockTasks.find((t: any) => t[column] === value)
+          } else if (table === 'vibes') {
+            data = mockVibes.find((v: any) => v[column] === value)
+          }
+
+          return { data, error: data ? null : { code: 'PGRST116' } }
+        },
+
+        order: (column: string, options?: any) => ({
+          then: async (callback: any) => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)